src/components/LiquidityMining/Create/Steps/PreviewAndCreate/index.tsx

```diff
@@ -23,16 +40,55 @@ const ResponsiveContainer = styled(Box)<{ flex1?: boolean }>`
     margin-top: 16px !important;
   `}
 `
+const StyledCampaignCard = styled(CampaignCard)`
+  width: 50%;
+`
+const CampaignDetailWrapper = styled(Flex)`
+  gap: 32px;
+`
+const SimulatedValue = styled.div`
+  font-family: 'Fira Mono';
```

Comment:
This style is going to be deprecated as our app is moving to Inter font. I think we should define no font-family in here and let it be the app default.